subsys/bluetooth/host/hci_core.c

```diff
@@ -82,6 +82,7 @@ struct bt_dev bt_dev = {
 static bt_ready_cb_t ready_cb;
 
 static bt_le_scan_cb_t *scan_dev_found_cb;
+static sys_slist_t scan_cbs;
```

Comment:
Minor: this could be:
```
static sys_slist_t scan_cbs = SYS_SLIST_STATIC_INIT(&scan_cbs);
```

That way you don't need the runtime `sys_slist_init()` initialization. Actually, seems you had that call previously but removed it, or do I remember wrong? The end result doesn't really change here since sys_slist_t just contains two pointers that should be initialised to zero (NULL), however for correctness the initialisation should be there (in case e.g. the underlying implementation changes to something more complex in the future).